### DISCUSSION TITLE
No Config

### DISCUSSION THREAD

**Discussion by RA3TOX** [dl9rdz/rdz_ttgo_sonde#279] [Q&A]
My equipment is HELTEC ESP32 Lora v2. 
I uploaded the firmware (devel20220123-B16-full.bin). 
The device is working. But I only see empty fields in the config settings. 
What could be the problem?
![rdzTTGO-noconfig](https://user-images.githubusercontent.com/67674698/153549048-9df8a5c6-f9e6-49e8-9f2a-5974e8959ec2.jpg)

**Comment by dl9rdz** [comment]
The only way I know this happens is if you did an online update from an older version., and this can be resolved by doing the update a second time.

Also you need JavaScript enabled in your browser.

**Reply by RA3TOX** [reply]
I installed the program only for the first time on a new Heltec board. 
And I'm trying to set the configuration from a smartphone.

**Comment by RA3TOX** [comment]
Thank you. I found everything. Launched from a PC and saw everything! 
Today I will test with the MRZ meteo-ballon.
